scripts/ci-ai-model-serving/tests/05-test-cuda-vector-add.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+oc create namespace cuda-test
+
+cat << EOF | oc apply -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test-cuda-vector-add
+  namespace: cuda-test
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: cuda-vector-add
+        image: "nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0-ubi8"
+        resources:
+          limits:
+            nvidia.com/gpu: 1
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop: ["ALL"]
+          runAsNonRoot: true
+          seccompProfile:
+            type: "RuntimeDefault"
+EOF
+
+# Wait till complete
+oc wait --for=condition=complete --timeout=120s -n cuda-test job/test-cuda-vector-add
+
+# Get the logs
+pod=$(oc get pods -n cuda-test --selector=batch.kubernetes.io/job-name=test-cuda-vector-add --output=jsonpath='{.items[*].metadata.name}')
+logs=$(oc logs -n cuda-test "${pod}")
+
+oc delete job -n cuda-test test-cuda-vector-add
+oc delete ns cuda-test
+
+if ! echo "${logs}" | grep -q PASSED; then
```

Comment:
Could we make it more precise, i.e. ^PASSED or like?

Reply:
This is the output.
```
[Vector addition of 50000 elements]
Copy input data from the host memory to the CUDA device
CUDA kernel launch with 196 blocks of 256 threads
Copy output data from the CUDA device to the host memory
Test PASSED
Done
```

Would `^Test PASSED$` be better?